requirements-dev.txt

```diff
@@ -15,7 +15,7 @@ cryptography==3.2.1       # via secretstorage
 distlib==0.3.0            # via virtualenv
 docutils==0.16            # via readme-renderer
 filelock==3.0.12          # via virtualenv
-flake8==3.5.0             # via -r requirements-dev.in
+flake8==3.8.0             # via -r requirements-dev.in
```

Comment:
this file is auto-generated based on the input in
https://github.com/Netflix/lemur/blob/7ea6e2059a6cacf4284cf901de26ab335d6775f2/requirements-dev.in#L3

once the requirement is modified, one can either run 
```
make up-reqs
```
to compute the requirements-dev.txt

or delegate to dependbot to file a PR

Reply:
Thanks for the details, flake8 update needed more version update and lint fixes. So I will do the manual steps and update the PR.